organizations/models.py

```diff
@@ -82,20 +83,31 @@ class Meta(object):
         verbose_name_plural = _('Link Courses')
 
 
+class UOMFiltersQuerySet(QuerySet):
+    def active(self):
+        return self.filter(is_active=True, organization__active=True)
+
+UOMManager = UOMFiltersQuerySet.as_manager
```

Comment:
I haven't edited object managers before, so I gotta ask.

Apart from the new `objects.active()` method, does this add anything else?

As you know we'd like to keep seeing those objects in LMS `/admin` panel.